docs/how-to/deploy-to-heroku.md

```diff
@@ -0,0 +1,80 @@
+# Introduction
+
+In this guide, we will be looking at how to deploy bocadillo powered app to Heroku.
+You can see example code in [heroku-example](https://github.com/bocadilloproject/heroku-example) repo.
+
+### Bocadillo Application
+
+Assuming, you have `app.py` already installed and it may look similar to this:
+
+```
+from bocadillo import App
+
+app = App()
+
+@app.route("/")
+async def index(req, res):
+    res.text = "Hello, from Heroku!"
+
+
+if __name__ == "__main__":
+    app.run()
+```
+
+Make sure it is running in local machine.
+
+```
+python app.py
+# Started server process
+# Waiting for application startup.
+# Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
+```
+
+### Procfile
+
+A text file in the root directory of your application, to explicitly declare what command should be executed to start your app.
+
+```
+web: gunicorn -w 4 -k uvicorn.workers.UvicornWorker app:app
+```
+
+### Requirements.txt
+
+Heroku recognizes an app as a Python app by the existence of a Pipfile or requirements.txt file in the root directory. So mention all your python dependencies with the version in this file.
```

Comment:
Are you sure Heroku can recognise `Pipfile`? I wasn't able to find reference to that in the Heroku docs.

```suggestion
Heroku recognises a Python app by the existence of a Pipfile or requirements.txt file in the root directory (see [Python Dependencies via Pip](https://devcenter.heroku.com/articles/python-pip)). Here's an example of what yours may look like:
```

Reply:
I saw it in README of https://github.com/heroku/heroku-buildpack-python but yeah, when. I tried `Pipfile` it didn't recognize it though. I am going to remove it right now.